### ISSUE TITLE
flux bootstrap gitlab: add --hostname example

### ISSUE THREAD

**Issue by exokortex** [fluxcd/flux2#1654]
The documentation should clarify that this command is only for use with https://gitlab.com and not for self hosted GitLab instances.

**Comment by dholbach** [comment]
Probably needs fixing in `cmd/flux/bootstrap_gitlab.go`.

**Comment by stefanprodan** [comment]
There is a `hostname` in example in https://fluxcd.io/docs/cmd/flux_bootstrap_gitlab/ Maybe we should rephrase it to self-hosted GitLab server?

**Comment by exokortex** [comment]
I think it is fine as it is. I do not know how I did not see that.
